src/web/app/pages-student/student-profile-page/student-profile-page.component.ts

```diff
@@ -11,21 +11,13 @@ import { AuthInfo, MessageOutput, Nationalities } from '../../../types/api-outpu
 import { FormControl, FormGroup } from '@angular/forms';
 
 import { StatusMessageService } from '../../../services/status-message.service';
+import { StudentProfileService } from '../../../services/student-profile.service';
+import { StudentProfileUpdateRequest } from '../../../types/api-request';
 import { Gender } from '../../../types/gender';
 import { ErrorMessageOutput } from '../../error-message-output';
 
-interface StudentProfile {
-  shortName: string;
-  email: string;
-  institute: string;
-  nationality: string;
-  gender: Gender;
-  moreInfo: string;
-  pictureKey: string;
-}
-
 interface StudentDetails {
-  studentProfile: StudentProfile;
+  studentProfile: StudentProfileUpdateRequest;
```

Comment:
`StudentProfileUpdateRequest` -> `StudentProfile`?

Reply:
It is generated based on java request, will this break the convention? Since I saw every request DTO ends with `request`